src/js/state/migrations/202103011928_adjustWindowsNetworkingActivityQuery.ts

```diff
@@ -0,0 +1,28 @@
+import {getAllStates} from "../../test/helpers/getTestState"
+
+export default function adjustWindowsNetworkingActivityQuery(state: any) {
+  const shippedQuery = {
+    id: "3",
+    name: "Windows Networking Activity",
+    value: "_path=~smb* OR _path=dce_rpc",
+    description:
+      "Filters and displays smb_files, smb_mapping and DCE_RPC activity",
+    tags: ["windows", "smb", "malware"]
+  }
+
+  for (const s of getAllStates(state)) {
+    if (!s.queries) continue
+    const targetQuery = Object.values(s.queries.items).find(({id}) => {
+      return id === "3"
+    })
+    if (
+      targetQuery &&
+      JSON.stringify(targetQuery) === JSON.stringify(shippedQuery)
```

Comment:
I was curious, so I looked it up. It appears `JSON.stringify` does not order object keys the same way every time. If the two objects are equal but have keys in different orders, then it might not match. It'd be more reliable to use `Lodash.isEqual()` in this case.

The quote from MDN:

> Note: Properties of non-array objects are not guaranteed to be stringified in any particular order. Do not rely on ordering of properties within the same object within the stringification.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify